test/subset/meson.build

```diff
@@ -36,6 +36,7 @@ tests = [
   'colr_with_components',
   'cbdt',
   'variable',
+  'glyph_names',
```

Comment:
You need to add this to the relevant `Makefile.sources` to fix the distcheck failure.

Reply:
Done, thank you!